Src/AutoFixture/Kernel/ParameterSpecification.cs

```diff
@@ -11,6 +12,7 @@ public class ParameterSpecification : IRequestSpecification
     {
         private readonly Type targetType;
         private readonly string targetName;
+        private readonly IEqualityComparer<string> nameComparison;
```

Comment:
Given that the containing class is a `ParameterSpecification`, what is it going to compare?